Topics/Tech_Stacks/Supabase.md

```diff
@@ -0,0 +1,111 @@
+# Using Supabase with Vue.js
+
+[Supabase](https://supabase.com) is an open source [Firebase](https://firebase.google.com/) alternative for building secure and high-performance PostgreSQL backends. It provides developers with a wide array of tools, such as real-time database configuration, authentication, and storage.
+
+### Why use Supabase?
+
+- As an open-source product, Supabase grants flexibility and control over your data and applications.
+- Supabase provides an excellent UI that gives you a centralized view of your application’s data, authentication settings, APIs, etc. The dashboard is very user-friendly, which gives a great deal of control to developers of all experience levels.
+- Supabase provides [extensive documentation](https://supabase.com/docs) to help you get started with the platform and make the most of its features.
+
+### About this Guide
+
+This document will include
+
+- how to set up a basic [Vue.js](https://vuejs.org/) project
+- installing Supabase into your Vue.js project
+- an example of creating a small database using the Supabase dashboard
+
+## Prerequisites
+
+Before reading this guide, ensure you have [Node.js](https://nodejs.org/en/) and [npm](https://www.npmjs.com/) installed on your machine.
+
+Also, this guide is written with the assumption that the reader has a basic understanding of Vue and PostgreSQL.
+
+## Step 1: Setting Up a Basic Vue.js Project
+
+1. **Create a Vue Project**: You can use the command line interface to create a new Vue.js project.
+
+   ```bash
+   npm init vue@latest vue-app
+   ```
+
+2. **Navigate to the Directory**: Move into your newly initialized directory.
+   ```bash
+   cd vue-app
+   ```
+
+## Step 2: Installing Supabase JS Client
+
+1. **Install the Supabase Client**: Use npm again to install the Supabase JavaScript client.
+
+   ```bash
+   npm install @supabase/supabase-js
+   ```
+
+2. **Retrieve your Supabase API Keys**: Access your Supabase project dashboard and retrieve your Supabase URL and API keys (the dashboard is explained later).
```

Comment:
It would be better to give a visual example by taking screenshots of the Supabase interface and giving the user a walkthrough to get the API key.
(Also the dashboard is not explained later.)

Thus as a solution, we can either link to Supabase's existing tutorial on this topic, or provide a basic one here (or both).

Reply:
Added a screenshot of the dashboard for clarity.

Reply:
The screenshot is not loaded for me unfortunately, i would suggest just downloading the image, upload it as part of the repo and adding a relative link to the image in the markdown.